docs/src/main/asciidoc/_project-features-wiremock.adoc

```diff
@@ -63,6 +63,12 @@ NOTE: Actually, WireMock always loads mappings from `src/test/resources/mappings
 well as* the custom locations in the `stubs` attribute. To change this behavior, you can
 also specify a files root, as described in the next section of this document.
 
+NOTE: Also, the mappings in the `stubs` location are not considered part of Wiremock's "default mappings" and calls
+to `com.github.tomakehurst.wiremock.client.WireMock.resetToDefaultMappings` during a test will not result in the mappings
+in the `stubs` location to be included. However, the `org.springframework.cloud.contract.wiremock.WireMockTestExecutionListener`
+does reset the mappings (including adding the ones from the stubs location) after every test class and optionally
+after every test method (guarded by the `wiremock.reset-mappings-after-each-test` property).
```

Comment:
Should we link to the properties doc?

Reply:
What I try to do is to link to a test that has this property. If that's not possible then I'd leave it as it is.

Reply:
I think in this case because its just a simple straight forward property, its fine as it is. They either set that to true/false no need to be more explicit.